service/history/tasks/category.go

```diff
@@ -89,6 +91,12 @@ var (
 		cType: CategoryTypeImmediate,
 		name:  CategoryNameVisibility,
 	}
+
+	CategoryArchival = Category{
```

Comment:
Add this category to `categories` on L103 as well.

Reply:
Ah, thanks! Done

Reply:
Actually, adding this means we need to have a queue factory registered for this category, which I'm only planning on adding in a later diff, so I'd prefer to keep it out of this map for now.